spring-data-relational/src/main/java/org/springframework/data/relational/core/sql/SqlIdentifier.java

```diff
@@ -77,9 +78,12 @@ public String toString() {
 	 * Return the reference name after applying {@link IdentifierProcessing} rules. The reference name is used for
 	 * programmatic access to the object identified by this {@link SqlIdentifier}.
 	 *
+	 * @deprecated Use the toSql(IdentifierProcessing processing) method instead.
```

Comment:
mind the formatting, `@deprecated` comes after `@return`, we should also include that the deprecation will be since `3.1` in place (`@deprecated since 3.1, use the #getReference() method instead.`)

Reply:
Note that the replacement is `getReference()` not `toSql(IdentifierProcessing)`